index.html

```diff
@@ -3,10 +3,203 @@
+      <div class="pets-button">
+        <a href="#" title="Get to know the rest">Get to know the rest</a>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="container">
+      <div class="help-content">
+        <h3>How you can help our shelter</h3>
+        <div class="help-icon">
+          <figure>
+            <p><img src="img/icon-pet-food.png" alt="Pet food" /></p>
+            <figcaption>Pet food</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/transportation.png" alt="Transportation" /></p>
+            <figcaption>Transportation</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/toys.png" alt="Toys" /></p>
+            <figcaption>Toys</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/bowls-and-cups.png" alt="Bowls and cups" /></p>
+            <figcaption>Bowls and cups</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/shampoos.png" alt="Shampoos" /></p>
+            <figcaption>Shampoos</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/vitamins.png" alt="Vitamins" /></p>
+            <figcaption>Vitamins</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/medicines.png" alt="Medicines" /></p>
+            <figcaption>Medicines</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/collars-or-leashes.png" alt="Collars / leashes" /></p>
+            <figcaption>Collars / leashes</figcaption>
+           </figure>
+           <figure>
+            <p><img src="img/sleeping-area.png" alt="Sleeping areas" /></p>
+            <figcaption>Sleeping areas</figcaption>
+           </figure>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="backdonation">
+    <div class="container">
+      <div class="donation">
+        <div>
+          <img src="img/donation-dog.png" alt="Donation">
+        </div>
+        <div class="donation-content">
+          <h3>You can also make a donation</h3>
+          <p>Name of the bank / Type of bank account</p>
+          <div class="credit-card">
+            <img src="img/credit-card.png" title="credit card">
+            <span>8380 2880 8028 8791 7435</span>
+          </div>
+          <p class="legal-information">Legal information and lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas a ipsum at libero sagittis dignissim sed ac diam. Praesent ultrices maximus tortor et vulputate. Interdum et malesuada fames ac ante ipsum primis in faucibus.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer>
+    <section class="containerHeaderFooter">
+      <div class="footer">
+        <div class="footercolom">
+        <h3>For questions and suggestions</h3>
+        <div class="text_address">
+          <svg width="40" height="26" viewBox="0 0 40 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Check all `svg` in footer and align them with the line, for example, you can use `vertical-align: middle`